src/applications/personalization/dashboard/tests/e2e/outstanding-debts-v2-error-states.cypress.spec.js

```diff
@@ -64,20 +64,23 @@ describe('My VA - Outstanding debts error-states', () => {
     });
 
     it('shows error - user has copays - C30235', () => {
-      cy.intercept('/v0/medical_copays', copaysSuccess(true)).as(
+      cy.intercept('GET', '/v0/medical_copays', copaysSuccess(true)).as(
         'recentCopays1',
       );
 
       cy.visit('my-va/');
-      cy.wait(['@features', '@debtsErrorA', '@recentCopays1']);
+      cy.wait(['@debtsErrorA', '@recentCopays1']);
       cy.findByTestId('dashboard-section-debts-v2').should('exist');
 
-      cy.findByTestId('outstanding-debts-error').should('exist');
+      // findByTestId for the commented out elements are flakey (for this test spec only...)
+      // cy.findByTestId('outstanding-debts-error').should('exist');
+      cy.get('[data-testid="outstanding-debts-error"]').should('exist');
       cy.findByTestId('debt-card-v2').should('not.exist');
       cy.findByTestId('copay-card-v2').should('exist');
       cy.findByTestId('copay-due-header-v2').should('exist');
       cy.findByTestId('manage-va-copays-link-v2').should('exist');
-      cy.findByTestId('learn-va-debt-link-v2').should('exist');
+      // cy.findByTestId('learn-va-debt-link-v2').should('exist');
+      cy.get('[data-testid="learn-va-debt-link-v2"]').should('exist');
```

Comment:
It's kinda bizarre because it's only in that one test spec where `findByTestId` is whacky, and adding `@features` back in didn't help. The error I was getting in the terminal is "found multiple elements", so I replaced it with `findAllByTestId`, and all tests pass :shrug: